style.css

```diff
@@ -0,0 +1,1399 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+h1 {
+  color: rgb(160, 59, 59);
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  background-image: url('img/backgrounds/header.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.mobileMenu {
+  margin-top: 5%;
+  margin-left: 7%;
+}
+
+.heading {
+  position: fixed;
+}
+
+.logoText {
+  font-size: 14px;
+  color: #272a31;
+}
+
+#btnImg,
+#abtnImg {
+  width: 20px;
+  cursor: pointer;
+}
+
+.txtContainer {
+  display: flex;
+  flex-direction: column;
+  margin-top: 10%;
+  margin-left: 7%;
+  margin-right: 7%;
+}
+
+.txtGreeting {
+  display: flex;
+  flex-direction: row;
+  font-weight: 500;
+  font-size: 24px;
+  color: #ec5242;
+}
+
+.textTitle {
+  display: flex;
+  flex-direction: row;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  background-image: url('img/backgrounds/textbg.jpg');
+  background-clip: border-box;
+  background-repeat: no-repeat;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.txtParagraph {
+  display: flex;
+  flex-direction: row;
+  margin-top: 8px;
+  padding: 16px;
+  border: 2px solid #fff;
+}
+
+.txtParagraph p {
+  display: flex;
+  flex-direction: row;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  color: rgba(33, 33, 33, 0.5);
+}
+
+.dateContainer {
+  padding: 20px 0 0 0;
+}
+
+.txtDate {
+  display: flex;
+  flex-direction: row;
```

Comment:
- [Optional] I noticed that you have these two properties repeated all over your CSS. You should consider creating a class for it and assigning the class to the HTML elements rather than repeating the same properties. This will make your code neater and help you follow the [DRY principle.](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#:~:text=The%20DRY%20principle%20is%20stated,their%20book%20The%20Pragmatic%20Programmer.)